README.md

```diff
@@ -54,6 +54,8 @@ The default, pre-installed backend for Qadence is [PyQTorch](https://github.com/
 * `pulser`: The [Pulser](https://github.com/pasqal-io/Pulser) backend for composing, simulating and executing pulse sequences for neutral-atom quantum devices.
 * `braket`: The [Braket](https://github.com/amazon-braket/amazon-braket-sdk-python) backend, an open source library that provides a framework for interacting with quantum computing hardware devices through Amazon Braket.
 * `visualization`: A visualization library to display quantum circuit diagrams.
+* `protocols`: A collection of [protocols](https://github.com/pasqal-io/qadence-protocols) for error mitigation in Qadence.
+* `libs`: A collection of [functionalities](https://github.com/pasqal-io/qadence-libs) for graph machine learning problems build on top of Qadence.
```

Comment:
is it just graph ML stuff?

Reply:
No but that is the intent in the near future. In the end, it will encompass all algos related to QML, Graph ML, optimisation, chemistry that would be built on top of Qadence.